database/migrations/2023_10_28_162444_create_attributes_table.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration {
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('attributes', function (Blueprint $table) {
+            $table->char('id', 36)->collation('ascii_bin')->primary();
+            $table->char('facility_id', 36)->collation('ascii_bin')->nullable();
+            $table->char('table');
```

Comment:
kolumny uuid robię char naiwnie wierząc, że sql sobie je lepiej zoptymalizuje

poza tym, to dla typowych danych to raczej powinno być `string` wszędzie

może jakieś string o mniejszej długości, typu 36, dla `table`, `model`, `type` i `requirement_level`

a dla tych pól oraz `api_name` by mogły być `collation('ascii_bin')` w sumie